### PR TITLE
Front end pull request template

### PR DESCRIPTION
**Pull request by nguyenaiden**
***Description***: Currently, there is no standardized pull request template. Going forward, please follow this format when creating a pull request because it will allow the reviewers to have a clear idea of what the changes being shipped. Also, this will be very useful in the case where we need to go back to previously closed/merged PRs to see what was changed.

### CODE REVIEW

**Review by hammeiam — APPROVED** [review]
smh not even using your own template 🤪

**Review by nkrulikovsky — APPROVED** [review]

**Review by ug02fast — COMMENTED** [review]
Overall I like having a template. Particularly the description portion with key details to include in the PR.

The checklist I'm not too excited about. It's something that can live in a Contributing guide and be referenced as many of those checklist items are automated. I'm also more inclined to reduce the amount of actions devs should take that are manual and move them to automated processes if possible.

The template should be something someone outside of our direct team could successfully use to make an open source contribution if need be.

Curious to know your thoughts as well.

**Comment by hammeiam** [comment]
I agree with Art's comments. I'd also like to a reminder for reviewers in the template to explicitly review and ✅  any addresses included in the PR

File: PULL_REQUEST_TEMPLATE/deploy.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Please describe the issue/current behavior with as much context information as possible. 
+e.g: Currently, when a user does X, Y happens, and this leads to Z experience. So the solution here is to .....
+This is also a place to include before/after screenshots if applicable. Visual representation is preferred, but thorough written before/after comparison is also accepted.
+
+Fixes # (Add link to github issue if available)
+
+## Checklist before merging:
+
+- [ ] Branch is up-to-date with main branch of the main repo
+- [ ] All applicable Unit/Functional/E2E tests are included
```
[the remark this answers is not in the document]
Comment (nguyenaiden) [reply PULL_REQUEST_TEMPLATE/deploy.md:14]: Sounds good to me.

File: PULL_REQUEST_TEMPLATE/deploy.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- [ ] All current and new tests are passing (run `npm test`)
```
[the remark this answers is not in the document]
Comment (nguyenaiden) [reply PULL_REQUEST_TEMPLATE/deploy.md:15]: @ug02fast, after thinking it over, I agree with you on the checklist portion. I'll remove it and add necessary info into a Community Contribution Guide. @hammeiam, thoughts?

**Pull request merged**